src/main/java/cohen/worldcitysearch/servlet/WorldCitySearchServlet.java

```diff
@@ -0,0 +1,42 @@
+package cohen.worldcitysearch.servlet;
+
+import cohen.worldcitysearch.WorldCitiesList;
+import cohen.worldcitysearch.WorldCity;
+import com.google.gson.Gson;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class WorldCitySearchServlet extends HttpServlet
+{
+    private final WorldCitiesList citiesList = new WorldCitiesList();
+    private final Gson gson = new Gson();
+
+    public WorldCitySearchServlet() throws IOException
```

Comment:
This doesn't throw an exception

Reply:
It is required to throw an exception because the WorldCitiesList class throws IOException due to the file input being parsed and using the parse method
Should I be doing that differently?